eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver.php

```diff
@@ -273,7 +273,7 @@ private function warnAboutTooEarlyLoadedParams()
                 // Symfony 3.4+: . (PHP_SAPI === 'cli' ? 'make the affected commands lazy, ' : '')
                 . 'make the service lazy or see if you can inject another lazy service.',
                 $blame,
-                '"$' . implode(array_unique($params), '$", "$') . '$"'
+                '"$' . implode('$", "$', array_unique($params)) . '$"'
```

Comment:
@andrerom this is interesting - a result of `implode_call` CS fixer rule. Apparently it works the same regardless of the order of arguments (:man_facepalming:), however it's recommended to use documented ones. I've tested this, but letting you know just in case ;)

Reply:
This was indeed a mistake, so fix there is good 👍